app/main/menu.ts

```diff
@@ -1,15 +1,15 @@
 'use strict';
-const path = require('path');
+import * as path from 'path';
 
-const { app, shell, BrowserWindow, Menu, dialog } = require('electron');
+import { app, shell, BrowserWindow, Menu, dialog } from 'electron';
 
-const fs = require('fs-extra');
-const AdmZip = require('adm-zip');
-const { appUpdater } = require('./autoupdater');
+import * as fs from 'fs-extra';
+import * as AdmZip from 'adm-zip';
+import { appUpdater } from './autoupdater';
 
-const ConfigUtil = require(__dirname + '/../renderer/js/utils/config-util.js');
-const DNDUtil = require(__dirname + '/../renderer/js/utils/dnd-util.js');
-const Logger = require(__dirname + '/../renderer/js/utils/logger-util.js');
+import { ConfigUtil } from '../renderer/js/utils/config-util';
+import DNDUtil from '../renderer/js/utils/dnd-util';
```

Comment:
{ DNDUtil } ?

Reply:
That hasn't been migrated to typescript for now. So, for the sake of testing, left it as it was before. Will change when i migrate DNDUtil to typescript.

Reply:
Okay, sounds good.